sql/src/main/java/io/crate/execution/engine/profile/NodeCollectProfileResponse.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.execution.engine.profile;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.transport.TransportResponse;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+public class NodeCollectProfileResponse extends TransportResponse {
+
+    private Map<String, Long> durationByPhase = Collections.emptyMap();
```

Comment:
what is saved here as the String key?  its somehow not obvious to me... I guess it won't  be the phase name as one node can run multiple phases with the same name inside one plan...

Reply:
hm, you're right, the naming of the map is not 100 percent accurate. the keys are `phaseId + phaseName` so they are unique per job execution context. however, you could theoretically use any string as key for the timing.

Reply:
what about `durationByPhaseIdent` as property name?

Reply:
strictly speaking it is even the id + name of the context ;)

Reply:
yes that naming would help. if its not documented/obvious (maybe i missed something) it may make sense to add a short comment about that somewhere